src/docs/cookbook-manage/implement-delete-user/index.md

```diff
@@ -0,0 +1,574 @@
+---
+title: Implement "Delete account" functionality
+slug: implement-delete-user
+updated: 2022-03-25
+category: cookbook-manage
+ingress:
+  This article guides you in implementing a feature where a user can delete
+  their own account through the FTW template Account settings section.
+published: true
+---
+
+Starting from 2022-03, Sharetribe Flex Marketplace API has an endpoint
+for
+[deleting a user's account](https://www.sharetribe.com/api-reference/marketplace.html#delete-user).
+After the user has been deleted, any transactions where they are a
```

Comment:
Do you think it could be a good idea to mention that you should update the SDK to the newest version before starting?

Reply:
Excellent point! I added it to the server/endpoint section of the cookbook